### PR TITLE
feature(wallet-mobile): new tx review stake pool details

### PR DESCRIPTION
**Pull request by banklesss**
##### Ticket

YV-90

### CODE REVIEW

File: apps/wallet-mobile/src/features/ReviewTx/common/ExplorerInfoLinks.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import {Space} from '../../../components/Space/Space'
+import {useSelectedWallet} from '../../WalletManager/common/hooks/useSelectedWallet'
+import {useStrings} from './hooks/useStrings'
+
+export const ExplorerInfoLinks = ({id, type}: {id: string; type: 'token' | 'pool'}) => {
+  const {styles} = useStyles()
+  const {wallet} = useSelectedWallet()
+  const strings = useStrings()
+
+  const handleOpenLink = async (direction: 'cardanoscan' | 'adaex') => {
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/ReviewTx/common/ExplorerInfoLinks.tsx]:
```suggestion
  const handleOpenLink = async (explorer: Explorers.Explorer) => {
```

File: apps/wallet-mobile/src/features/ReviewTx/common/ExplorerInfoLinks.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    if (id == null) return
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/ReviewTx/common/ExplorerInfoLinks.tsx:16]: it can't be null

Comment (stackchain) [inline apps/wallet-mobile/src/features/ReviewTx/common/ExplorerInfoLinks.tsx:16]:
```suggestion
```

File: apps/wallet-mobile/src/features/ReviewTx/common/ExplorerInfoLinks.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    if (direction === 'cardanoscan') {
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/ReviewTx/common/ExplorerInfoLinks.tsx]:
```suggestion
    if (explorer === Explorers.Explorer.CardanoScan) {
```

File: apps/wallet-mobile/src/features/ReviewTx/common/ExplorerInfoLinks.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  }
+
+  return (
+    <View>
+      <Space width="sm" />
+
+      <Text style={styles.label}>{strings.details}</Text>
+
+      <View style={styles.linkGroup}>
+        <TouchableOpacity onPress={() => handleOpenLink('cardanoscan')}>
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/ReviewTx/common/ExplorerInfoLinks.tsx]:
```suggestion
        <TouchableOpacity onPress={() => handleOpenLink(Explorers.Explorer.CardanoScan)}>
```

File: apps/wallet-mobile/src/features/ReviewTx/common/ExplorerInfoLinks.tsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+          <Text style={styles.link}>Cardanoscan</Text>
+        </TouchableOpacity>
+
+        <TouchableOpacity onPress={() => handleOpenLink('adaex')}>
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/ReviewTx/common/ExplorerInfoLinks.tsx]:
```suggestion
        <TouchableOpacity onPress={() => handleOpenLink(Explorers.Explorer.CExplorer)}>
```

File: apps/wallet-mobile/src/features/ReviewTx/common/ExplorerInfoLinks.tsx [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+  return (
+    <View>
+      <Space width="sm" />
+
+      <Text style={styles.label}>{strings.details}</Text>
+
+      <View style={styles.linkGroup}>
+        <TouchableOpacity onPress={() => handleOpenLink('cardanoscan')}>
+          <Text style={styles.link}>Cardanoscan</Text>
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/ReviewTx/common/ExplorerInfoLinks.tsx]: maybe export from explorers a explorers constant with {[Explorers.Explorer]: {name: string}} ?

File: apps/wallet-mobile/src/yoroi-wallets/hooks/index.ts [hunk 7]
```diff
@@ -1,7 +1,7 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import {walletChecksum} from '@emurgo/cip4-js'
 import {Certificate} from '@emurgo/cross-csl-core'
-import {PoolInfoApi} from '@emurgo/yoroi-lib'
+import {FullPoolIfo, PoolInfoApi} from '@emurgo/yoroi-lib'
```
Comment (stackchain) [inline apps/wallet-mobile/src/yoroi-wallets/hooks/index.ts]: it looks there is a typo in the lib

Comment (banklesss) [reply apps/wallet-mobile/src/yoroi-wallets/hooks/index.ts]: created a PR: https://github.com/Emurgo/yoroi-lib/pull/157

Comment (banklesss) [reply apps/wallet-mobile/src/yoroi-wallets/hooks/index.ts]: updated the version

**Review by michalszorad — APPROVED** [review]

**Pull request merged**